docs/enterprise-guide/sso-keycloak.md

```diff
@@ -0,0 +1,59 @@
+# Enabling SSO with Keycloak
+Keycloak is an open source platform that can be used as a user directory to save user data while acting as the IdP for single sign-on.
+
+
+## Working in the Keycloak console
+1. Go to the Keycloak admin console and sign in as an administrator.
+2. Create a test user from **Manage** > **Users**. You'll need to populate the fields with an email, first name, and last name.
+3. Once you've created at least one user, navigation tabs will appear at the top of the **Users** page. Go to **Credentials** to set password for your test user.
+  - Turn off the **Temporary** toggle. 
+  - Click **Set Password** to save your changes. 
+4. Create a new SSO client from **Manage** > **Clients** > **Create**.
+  - Client ID: Enter “metabase” in lowercase.
+  - Client Protocol: Select “saml” from the dropdown.
+  - Click **Save**.
+5. Configure the SSO client from the form that appears after saving:
+  - Client Signature Required: DISABLE
+  - Valid Redirect URIs: URL where you are hosting your Metabase instance followed by
+a slash (/) and an asterisk (*). For example, `http://localhost:3000/*`.
+  - Base URL: “URL the IdP should redirect back to” under Metabase **Admin settings** > **Authentication** > **<your SSO provider>**. For example, `“http://localhost:3000/auth/sso”`.
```

Comment:
This doesn't render well:

![image](https://user-images.githubusercontent.com/17343328/170501746-00254ad8-6214-4b41-a740-8420be8b962a.png)

I'm guessing it's the `<`s screwing up the markdown renderer.